public/styles.css

```diff
@@ -0,0 +1,184 @@
+body {
+    font-family: 'Poppins', sans-serif;
+    background: #1a1a2e;
+    color: #fff;
+    min-height: 100vh;
+    margin: 0;
+    padding: 40px 20px;
+    background-image: 
+        radial-gradient(circle at 10% 20%, rgba(91, 37, 255, 0.1) 0%, transparent 50%),
+        radial-gradient(circle at 90% 80%, rgba(255, 65, 108, 0.1) 0%, transparent 50%);
+}
+
+h1 {
+    font-size: 3.5rem;
+    text-align: center;
+    margin-bottom: 3rem;
+    background: linear-gradient(45deg, #5b25ff, #ff416c);
+    -webkit-background-clip: text;
+    background-clip: text;    -webkit-text-fill-color: transparent;
+    text-shadow: 2px 2px 20px rgba(91, 37, 255, 0.2);
+}
+
+#todo-form {
+    max-width: 800px;
+    margin: 0 auto 40px;
+    display: flex;
+    gap: 20px;
+    background: rgba(255, 255, 255, 0.03);
+    padding: 25px;
+    border-radius: 20px;
+    backdrop-filter: blur(10px);
+    border: 1px solid rgba(255, 255, 255, 0.1);
+}
+
+#todo-input {
+    flex: 1;
+    background: rgba(255, 255, 255, 0.05);
+    border: 2px solid rgba(255, 255, 255, 0.1);
+    padding: 15px 25px;
+    border-radius: 15px;
+    color: #fff;
+    font-size: 1.1rem;
+    transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+#todo-input:focus {
+    border-color: #5b25ff;
+    box-shadow: 0 0 25px rgba(91, 37, 255, 0.2);
+    outline: none;
+}
+
+button {
+    background: linear-gradient(45deg, #5b25ff, #ff416c);
+    color: white;
+    border: none;
+    padding: 15px 30px;
+    border-radius: 15px;
+    font-weight: 600;
+    font-size: 1rem;
+    cursor: pointer;
+    transition: transform 0.3s cubic-bezier(0.4, 0, 0.2, 1);
+    box-shadow: 0 5px 15px rgba(91, 37, 255, 0.2);
+}
+
+button:hover {
+    transform: translateY(-3px) scale(1.05);
+    box-shadow: 0 8px 25px rgba(91, 37, 255, 0.3);
+}
+
+.todo-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+    gap: 30px;
+    max-width: 1200px;
+    margin: 0 auto;
+}
```

Comment:
Hi @alphineandre 

I think you can use "display: block" to ensure that the task text and the timestamp are displayed on separate lines. This will make the task appear on top, and the timestamp will be below it.

Reply:
```css
 .todo-content {
      flex: 1;
      display: flex;
      flex-direction: column;
      gap: 5px;
  }

  .todo-text {
      color: #fff;
      font-size: 1.1rem;
  }

  .todo-timestamp {
      color: rgba(255, 255, 255, 0.5);
      font-size: 0.8rem;
  }
``` 

Issue resolved.

Added the above parameters to make the task name and timestamp to be in different lines. 

Thank you for making me aware of this minor issue.

Reply:
perfect!